packages/components/button/README.mdx

```diff
@@ -0,0 +1,112 @@
+- Use the left button for the primary action, if the button group is left- or center-aligned
+- Reduce complexity by using a small number of actions. Too many actions can create confusion when having to decide
+
+## Component variations
+
+- **Primary** - Used for the most important actions in any scenario. Don’t use more than one primary button in a section or screen to avoid overwhelming users.
+- **Secondary** - Used for a secondary actions, the most commonly used button type.
+- **Positive** - For use when the action has a positive connotation such as creating or publishing a new entity.
+- **Negative** - For destructive actions - when something can't be undone. For example, deleting entities.
+
+```jsx
+<>
+  <Button variant="primary">Primary</Button>
+  <Button variant="secondary">Secondary</Button>
+  <Button variant="negative">Negative</Button>
+  <Button variant="positive">Positive</Button>
+</>
+```
+
+Buttons can be used in couple of different variations, like active, disabled or loading. Worth mentioning is `indicateDropdown` property in the component, which enables chevron icon on the right side of the button. Using `icon` property in the Button on the other hand, enables choosen icon on the left side of the button.
+
+Contentful buttons are available in 3 different sizes:
+
+```jsx
+<>
+  <Button variant="positive" size="large">
+    Large
+  </Button>
+  <Button variant="positive">Medium (Default)</Button>
+  <Button variant="positive" size="small">
+    Small
+  </Button>
+</>
+```
+
+## Code examples
+
+```tsx static=true
+import { Button } from '@contentful/f36-button';
+
+<Button variant="primary" testId="create-content-type-empty-state">
+  Add content type
+</Button>;
+```
+
+Usage of the button with icon
+
+```tsx static=true
+import { Button } from '@contentful/f36-button';
+import { Download } from '@contentful/f36-icons';
+
+<Button variant="primary" icon={Download}>
+  Primary star
+</Button>;
+```
+
+Icon Button
+
+```tsx static=true
+import { Button } from '@contentful/f36-button';
+import { Download } from '@contentful/f36-icons';
+
+<Button variant="primary" aria-label="Download Asset" icon={Download} />;
+```
+
+## Content recommendations:
+
+- UI usage - Button labels should be no longer than 3 words
+- Webpage usage - Button labels should be no longer than 5 words
+- Start labels with action verbs. If the button is part of an action dialog, make sure it matches the dialog header.
+- Use specific words, ideally ones that align with preceding content.
+- For buttons that are used to cancel destructive actions: label them "Never mind" or similar, instead of cancel. It makes things much easier for users to understand.
+
+## Accessibility
+
+Button component is used to trigger an action or event. By default button is implemented using the native `<button>` element with the proper type="button" provided. If the href element is provided, then the element would be rendered as `<a>` element in the document.
```

Comment:
README 🙌 ! Just a little comment: maybe we could make a list out of it, it would be probably rendered as one block of text.

Reply:
@burakukula you mean a list of the outline, and add it to the plop template?